License2Deploy/rolling_deploy.py

```diff
@@ -296,12 +321,13 @@ def get_args(): # pragma: no cover
   parser.add_argument('-a', '--ami', action='store', dest='amiID', help='AMI ID to be deployed', type=str, required=True)
   parser.add_argument('-P', '--profile', default='default', action='store', dest='profile', help='Profile name as designated in aws credentials/config files', type=str)
   parser.add_argument('-c', '--config', default='/opt/License2Deploy/regions.yml', action='store', dest='config', help='Config file Location, eg. /opt/License2Deploy/regions.yml', type=str)
+  parser.add_argument('-s', '--stack', action='store', dest='stack_name', help='Stack name if AutoScaling Group created via CloudFormation', type=str)
   return parser.parse_args()
 
 def main(): # pragma: no cover
   args = get_args()
   SetLogging.setup_logging()
-  deployObj = RollingDeploy(args.env, args.project, args.buildNum, args.amiID, args.profile, args.config)
+  deployObj = RollingDeploy(args.env, args.project, args.buildNum, args.amiID, args.profile, args.config, args.stack_name)
```

Comment:
Did we want to change buildNum here (and in get_args) to build_number to be consistent?